docs/api/portable-stories-jest.md

```diff
@@ -26,10 +26,11 @@ Normally, Storybok composes a story and its [annotations](#annotations) automati
 
 <Callout variant="info">
 
-**Using `Next.js`?** You need to do two things differently when using portable stories in Jest with Next.js projects:
+**Using `Next.js`?** You need to do three things differently when using portable stories in Jest with Next.js projects:
 
 - Configure the [`next/jest.js` transformer](https://nextjs.org/docs/pages/building-your-application/testing/jest#manual-setup), which will handle all of the necessary Next.js configuration for you.
 - Import [`composeStories`](#composestories) or [`composeStory`](#composestory) from the `@storybook/nextjs` package (e.g. `import { composeStories } from '@storybook/nextjs'`).
+- Set up [internal module aliases](../get-started/nextjs.md#storybooknextjsexport-mocks) to be able to mock and assert on them.
```

Comment:
I think ideally people should always set up the aliases if they can, because right now the aliases only refer to the mocks, but might be extended in the future so that it's required for people to use the aliases. For instance, when we end up supporting multiple Next.js versions where internal paths change based on versions.

Reply:
More info: https://github.com/storybookjs/storybook/pull/26688/files#diff-c1603ee6a674bb75255aa448057630506153e363297fd58668e6679f10909bc6R417

Reply:
Thanks for the info!